ios/browser/api/ads/brave_ads.h

```diff
@@ -148,6 +148,13 @@ OBJC_EXPORT
 
 - (void)clearData:(void (^)())completion;
 
+#pragma mark - New Tab Page Ad
+
+- (nullable NewTabPageAdIOS*)maybeGetPrefetchedNewTabPageAd;
+
+- (void)onFailedToPrefetchNewTabPageAd:(NSString*)placementId
```

Comment:
append `NS_SWIFT_NAME(onFailedToPrefetchNewTabPageAd(placementId:))` so it doesn't become `onFailed(toPrefetch…)` in Swift automatically

Reply:
Is it fine to follow-up, please? 
Just appending `NS_SWIFT_NAME(onFailedToPrefetchNewTabPageAd(placementId:))` gave me an error

Reply:
Follow-up is fine